.gitignore

```diff
@@ -0,0 +1,3 @@
+Gemfile.lock
```

Comment:
No it is not duplicate information. The file contains the exact version that you had installed at that certain point of time. It ensures package consistency. If someone else goes in and installs the gems he gets the ones with that exact same versions noted in the lock file instead of the most actual one matching your Gemfile definition. Removing the file could lead to unwanted behaviour because of inconsistent/unexpected environment.